include/grpc/grpc.h

```diff
@@ -387,7 +386,8 @@ grpc_call *grpc_channel_create_registered_call(
    completion of type 'tag' to the completion queue bound to the call.
    The order of ops specified in the batch has no significance.
    Only one operation of each type can be active at once in any given
-   batch. */
+   batch. 
+   THREAD SAFETY: grpc_call_start_batch is thread-compatible */
```

Comment:
I'd prefer to know that what C# is doing will be supported in the future. ;-)
Could you rephrase the restriction so I understand what you've actually meant? I can't guarantee at all from which thread the op will be call. Because the original docs said the call methods thread compatible I only initiate batched from within a lock owned by a call object, which is my understanding of how "thread-compatible" api method should be used.